### PR TITLE
Update Contributing file

### PR DESCRIPTION
**Pull request by Andre601**
### Type

- [ ] Internal change (Doesn't affect end-user).
- [ ] External change (Does affect end-user).
- [ ] Wiki (Changes towards the [Wiki]).
- [x] Other: CONTRIBUTING.md 

### Description

Since we apparently go away from the `development` branch and instead now only have the `master` branch did I quickly update the `CONTRIBUTING.md` file to reflect that change (And to hopefully make it a bit more understandable in some way).

Closes N/A 

[Wiki]: https://github.com/PlaceholderAPI/PlaceholderAPI/wiki

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -37,15 +38,30 @@ PlaceholderAPI provides a feature to have expansions (separate jar files) for pl
 In those cases should you report the issue to the issue tracker of the expansion or plugin.
 
 ## Pull requests
-As an open source project are we welcoming all contributions to improve PlaceholderAPI, being it changes to its code like bug fixes or new features, or contributions to its documentation such as the [Wiki] or the Javadoc.
+As an open source project are we welcoming all contributions to improve PlaceholderAPI, being it changes to its code, or contributions to its documentation such as the [Wiki] or the Javadocs.
+
+**When contributing, make sure to both base of and target the mentioned branch. Pull requests targeting the wrong branch may get closed without a warning.**
```
Comment (darbyjack) [inline .github/CONTRIBUTING.md:43]: This sentence right here doesn't feel like it flows properly. "Make sure to both base of and target the mentioned branch". Is "of" supposed to be "off"?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -37,15 +38,30 @@ PlaceholderAPI provides a feature to have expansions (separate jar files) for pl
...
 
 ### Code contributions
-Any contributions to PlaceholderAPI's code should be done towards the [`development`][dev] branch. Targeting the `master` branch in your Pull request may get it closed without warning.  
-Additionally should you follow the project's codestyle which you can find in the [`config/style`][style] directory.
+> **Source and Target Branch:** [`master`][master]
+
+When contributing towards the code of PlaceholderAPI, be it new features or just bug fixes, should your changes follow the general code styling used in the project.
```
Comment (darbyjack) [inline .github/CONTRIBUTING.md:48]: Last part of the sentence seems off. Maybe "your changes should follow the ..."?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -37,15 +38,30 @@ PlaceholderAPI provides a feature to have expansions (separate jar files) for pl
...
+
+Javadocs changes should usually be combined with [code contributions](#code-contributions) when possible, but if not, make sure the changes are significant enough to warrant a new build on our CI server.
+
+\*This branch may change in the future.
+
+### Wiki contributions
+> **Source and Target Branch:** [`docs/wiki`][docs-wiki]
+
+The Wiki of PlaceholderAPI has a unique quirk by having a dedicated [`wiki` folder][wiki] hosting the pages found on the actual wiki itself.  
+If you find outdated information, want to add missing expansions/plugins or just like to correct some wrong spelling should you both base of and target your contributions on the `docs/wiki` branch.
```
Comment (darbyjack) [inline .github/CONTRIBUTING.md:62]: Sentence flow here with the "should you both base of and target"

**Review by darbyjack — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (Andre601) [reply .github/CONTRIBUTING.md:43]: Idk... Not a native english speaker so I can't really tell which one is correct here...

In that sentence I basically want to say that the source and target branch should be the mentioned one.

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (Andre601) [reply .github/CONTRIBUTING.md:62]: See my other comment regarding this.

**Review by darbyjack — APPROVED** [review]

**Pull request merged**